src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -125,23 +174,36 @@ export default class ReplyField extends React.Component<Props, State> {
     };
 
     render(): JSX.Element {
-        const { className, isDisabled, placeholder, ...rest } = this.props;
+        const { className, isDisabled, itemRow, placeholder, ...rest } = this.props;
         const { editorState, popupReference } = this.state;
 
         return (
             <div className={classnames(className, 'ba-ReplyField')}>
-                <Editor
-                    {...rest}
-                    ref={this.editorRef}
-                    editorState={editorState}
-                    onChange={this.handleChange}
-                    placeholder={placeholder}
-                    readOnly={isDisabled}
-                    stripPastedStyles
-                    webDriverTestID="ba-ReplyField-editor"
-                />
-
-                {popupReference && <PopupList reference={popupReference} />}
+                <PopupList
+                    onSelect={this.handleSelect}
+                    reference={popupReference}
+                    selector={
+                        <Editor
+                            {...rest}
+                            ref={this.editorRef}
+                            editorState={editorState}
+                            onChange={this.handleChange}
+                            placeholder={placeholder}
+                            readOnly={isDisabled}
+                            stripPastedStyles
+                            webDriverTestID="ba-ReplyField-editor"
+                        />
+                    }
+                >
+                    {popupReference &&
+                        this.getItemList(editorState).map(item =>
+                            React.cloneElement(itemRow as JSX.Element, {
+                                ...itemRow!.props, // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
I think we should avoid disabling this rule. Can we structure our components in a way to avoid interacting with their props?